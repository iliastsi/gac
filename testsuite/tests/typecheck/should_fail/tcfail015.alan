scopError() : proc
  fun() : proc
  { foo = 5; }

  foo : int;
{}
