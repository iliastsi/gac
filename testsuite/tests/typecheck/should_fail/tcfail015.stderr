tcfail015.alan:3:5-7: Error: Not in scope `foo'
	Each undeclared identifier is reported only once for each function it appears in

